app/src/shared/components/SvgIcon/index.jsx

```diff
@@ -47,6 +49,9 @@ const sources = {
             />
         </svg>
     ),
+    imageUpload: (
+        <UploadIcon />
```

Comment:
Maybe we should put the raw svg element here?

Reply:
I was thinking that but finally I just lifted the whole component from Marketplace. The markup was bigger than the other, this seemed cleaner 🙄

Reply:
Ah, true. Looks pretty big. Maybe it's better this way 👍